content/contribution-guide/index.html

```diff
@@ -71,6 +69,16 @@
         </li>
         <li><a href="/blog">Blog</a></li>
       </ul>
+      <ul class="nav navbar-nav navbar-right">
+      <li class="dropdown">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project <span class="caret"></span></a>
+        <ul class="dropdown-menu">
+          <li><a href="/presentation-materials/">Presentation Materials</a></li>
+          <li><a href="/material/">Material</a></li>
+          <li><a href="http://apache.org/licenses/LICENSE-2.0.html">License</a></li>
+        </ul>
+      </li>
+    </ul>
```

Comment:
(closes place I can leave a comment on):

I'd rename `Apache-Beam - Project-Palette.pdf` to `palette.pdf`. Shorter, lower cases, no repetition, no spaces, no special characters. Similarly, `beam-logos.zip` could be `logos.zip`. In general, I'd try to be consistent in naming of files.